web/src/utils/formatting.ts

```diff
@@ -19,18 +18,37 @@ export const formatPower = function (
   return power;
 };
 
-export const formatCo2 = function (d: number, numberDigits: number = DEFAULT_NUM_DIGITS) {
-  let value = d;
+export const formatCo2 = function (gramPerHour: number, valueToMatch?: number) {
```

Comment:
This feels a little complext in relation to the scale power function we have below. Do we really need all the d3 format stuff instead of just rounding dividing, rounding and returning the unit ourselves. We already do return the units manually.